e2e/cypress/test_module/cypress/support/step_definitions/auth.steps.ts

```diff
@@ -1,4 +1,4 @@
-/// <reference types="Cypress" />
+/// <reference types="cypress" />
```

Comment:
Use uppercase `Cypress`, because it threw error for me

Reply:
Fixed in tsconfig